SwiftCompilerSources/Package.swift

```diff
@@ -15,6 +15,7 @@ import PackageDescription
 
 private extension Target {
   static let defaultSwiftSettings: [SwiftSetting] = [
+    .interoperabilityMode(.Cxx),
     .unsafeFlags([
       "-Xfrontend", "-validate-tbd-against-ir=none",
```

Comment:
you can remove these flags, they're set now for interop.

Reply:
Thanks @hyp!